contentctl/output/conf_writer.py

```diff
@@ -61,7 +62,7 @@ def writeManifestFile(app_output_path:pathlib.Path, template_name : str, config:
         j2_env = ConfWriter.getJ2Environment()
         template = j2_env.get_template(template_name)
         
-        output = template.render(objects=objects, APP_NAME=config.app.label, currentDate=datetime.datetime.now(datetime.UTC).date().isoformat())
+        output = template.render(objects=objects, app=config.app, currentDate=datetime.datetime.now(datetime.UTC).date().isoformat())
```

Comment:
Now pass the whole app to the jinja template so we can pull out extra relevant fields like the label

Reply:
LGTM